parsec/backend/memory/user.py

```diff
@@ -285,26 +275,33 @@ async def find_humans(
             )
             for user in data
         ]
-
+        results = sorted(
+            [res for res in results if res.human_handle], key=lambda r: r.user_id.lower()
+        )
         if omit_revoked:
             results = [res for res in results if not res.revoked]
 
         # PostgreSQL does case insensitive sort
-        humans = sorted(
-            [res for res in results if res.human_handle],
-            key=lambda r: (r.human_handle.label.lower(), r.user_id.lower()),  # type: ignore
-        )
-        non_humans = sorted(
-            [res for res in results if not res.human_handle], key=lambda r: r.user_id.lower()
-        )
-
-        if omit_non_human:
+        if omit_non_human or query:
+            total = len(results)
```

Comment:
`total` and the paginated `results` can be put before the if

Reply:
done